fmriprep/info.py

```diff
@@ -62,7 +61,8 @@
 
 LINKS_REQUIRES = [
     'git+https://github.com/nipy/nipype.git@master#egg=nipype',
-    'git+https://github.com/incf/pybids.git@master#egg=pybids'
+    'git+https://github.com/incf/pybids.git@master#egg=pybids',
+    'git+https://github.com/poldracklab/niworkflows.git@master#egg=niworkflows',
```

Comment:
For this to work, you need to keep the old ```'niworkflows'``` entry in the ```REQUIRES``` list. Make sure that there is no version selector though.